src/app/shared/helpers/item-route.ts

```diff
@@ -40,6 +53,17 @@ export function incompleteItemUrl(itemId: string): RouterCommands {
   return [ 'items', 'by-id', itemId ];
 }
 
+/*
+ * Url (as string) of the details page for the given item route
+ */
+export function linkToRoute(item: ItemRoute): string {
```

Comment:
shouldn't the name be `routeToLink` ? because the function returns a link from a route

Reply:
Ah I did not mean "convert a link to a route" but "(get) link which leads to a route" ;-)
I will find a better name...